explorer/scan/local_scanner.go

```diff
@@ -127,6 +127,27 @@ func (s *LocalScanner) RunIncognito(ctx context.Context, job *Job) (*explorer.Re
 	return reports, nil
 }
 
+// preprocessPolicyFilters expends short registry mrns into full mrns
+func preprocessQueryPackFilters(filters []string) []string {
+	res := make([]string, len(filters))
+	for i := range filters {
+		f := filters[i]
+		if strings.HasPrefix(f, "//") {
+			res[i] = f
+			continue
+		}
+
+		// expand short registry mrns
+		m := strings.Split(f, "/")
+		if len(m) == 2 {
```

Comment:
should just specifying the pack w/o namespace fallback to our (mondoohq?) namespace?
```
cnquery scan .... --querypack mondoo-okta-incident-response
```

Reply:
interesting idea. love it. lets do this in another iteration.